src/target.rs

```diff
@@ -305,9 +314,13 @@ impl Target {
                     format!("macosx_{}_{}_arm64", arm64_major, arm64_minor)
                 }
             }
+            // Windows
             (Os::Windows, Arch::X86) => "win32".to_string(),
             (Os::Windows, Arch::X86_64) => "win_amd64".to_string(),
             (Os::Windows, Arch::Aarch64) => "win_arm64".to_string(),
+            // Emscripten
+            // FIXME: should include emscripten version, example: html_py_ever-0.1.0-cp310-cp310-emscripten_3_1_13_wasm32.whl
```

Comment:
@hoodmane Is there any documentation about wheel naming in Pyodide?

Reply:
https://github.com/pyodide/pyodide/blob/38fae6e9cbcdd8e932a53bf1c763279dc55729d5/pyodide-build/pyodide_build/common.py#L15-L18

I think it's generated here in Pyodide, I wonder if there is any standard between different Python Emscripten implementations.

Reply:
We are going based on what the `packaging` package says:
```py
>>> import packaging.tags
>>> str(next(packaging.tags.sys_tags()))
'cp310-cp310-emscripten_3_1_13_wasm32'
```